composer.json

```diff
@@ -12,10 +12,8 @@
         }
     ],
     "require": {
-        "phpspec/prophecy": "~1.5"
-    },
-    "require-dev": {
-        "atoum/atoum": "~2.0"
+        "phpspec/prophecy": "~1.5",
+        "atoum/atoum": "^2.0"
```

Comment:
Moving atoum/atoum to the require section was part of the issue, see: https://github.com/atoum/prophecy-extension/issues/4